### DISCUSSION TITLE
WebView2 API for Azul Java JDK

### DISCUSSION THREAD

**Discussion by Fxztam** [MicrosoftEdge/WebView2Feedback#2416] [Tips and Tricks]
Hi,

I found WebView2 APIs for Python, GO and Flutter WebView2 and they work great on my Windows 10.
It would further contribute to the spread of WebView2 and it would also be a great step forward 
if there was a WebView2 for Java API e.g. for the Azul JDK!

The JDK 18 with the Panama project has a header file scanner for FFI API building and
might be possible to build the Java API wrapper for WebView2.
Are there any plans to provide this Java API wrapper for WebView2?

Thanks, Fried

**Comment by victorthoang** [comment]
Hey @Fxztam 

If you're still interested in this, would you mind if this gets reposted in the Issues thread with the question label? That way I can assign bring up the question to one of our leads for discussion.

If you're no longer interested, We can leave this as is, but typically this Discussions thread isn't actively monitored.
